src/routes/users/fetch-users.e2e.test.js

```diff
@@ -0,0 +1,57 @@
+const { expect } = require('chai')
+const { fixtures, setupDB } = require('../../../test/utils')
+
+describe('GET /users', () => {   
+	describe('Failure', ()=> {
+		let userUser
+		let accessToken
+		before(async () =>{
+			userUser = await fixtures.insertUser({
+				role : 'user'
+			}) 
+			const body = { id: userUser.id, email: userUser.email }
+			accessToken = fixtures.generateAccessToken(
+				{user : body}
+			)
+		})
+		it('should return 403 if user is not an admin', async() =>{
+			const expectedError = {
+				"status":"failed",
+				"message":"Only admin can create users"
+			}
+			fixtures.api()
+				.get(`/api/v1/users`)
+				.set('Authorization', `Bearer ${accessToken}`)
+				.expect(403 , expectedError)
+		})
+	}) 
+	describe('Success', () => {
+		let adminUser
+		let accessToken
+		before(async () =>{
+			await setupDB()
+			await fixtures.insertMultipleUsers()
```

Comment:
let's be explicit about the number of users being passed to the function like was done in the service test to help anyone reading the code understand what we're using in the `expect`

Reply:
great